### PR TITLE
added Login page and sign page

### PR DESCRIPTION
**Pull request by mikesch803**
It has both login and signup page on the same branch

### CODE REVIEW

File: pages/signup.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                <div class="navbar-icons">
+                    <a href="#" class="btn btn-link navbar-login">login</a>
+                    <a class="navbar-wishlist" href="#"><span class="iconify navbar-wishlist-icon"
+                            data-icon="mdi:cards-heart-outline" data-inline="false"></span></a>
+                    <a class="btn btn-link navbar-cart" href="#"><span class="iconify navbar-cart-icon"
+                            data-icon="zmdi:shopping-cart" data-inline="false"></span></a>
+                </div>
+            </div>
+        </header>
+        <div class="form">
```
Comment (sainath7878) [inline pages/signup.html]: Instead of using div you can use form tag.

File: pages/signup.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <h2 class="title-login">Signup</h3>
```
Comment (sainath7878) [inline pages/signup.html]: opening is a h2 tag but closing is a h3 tag.

File: pages/signup.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <input type="text" placeholder="email" class="form-input">
```
Comment (sainath7878) [inline pages/signup.html]: if input is only email, then you can give type = email

File: pages/signup.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            <input type="text" placeholder="password" class="form-input">
```
Comment (sainath7878) [inline pages/signup.html]: type must be password for a password

File: pages/signup.html (hunk 1 above)
Comment (mikesch803) [reply pages/signup.html]: changed to form tag

File: pages/signup.html (hunk 2 above)
Comment (mikesch803) [reply pages/signup.html]: thanks for point out the mistake

File: pages/signup.html (hunk 4 above)
Comment (mikesch803) [reply pages/signup.html]: added type password

File: pages/signup.html (hunk 3 above)
Comment (mikesch803) [reply pages/signup.html]: added type email

**Review by vrinda-mahajan — APPROVED** [review]

File: css/login.css [hunk 5]
```diff
@@ -0,0 +1,32 @@
+.grid-layout {
+    display: grid;
+    height: 100vh;
+    grid-template-areas: "header header header"
+                        "... ... ..."
+                        "... login ..."
+                        "... ... ..."
```
Comment (vrinda-mahajan) [inline css/login.css]: I think you forgot to add a semicolon to the grid template areas. Everything else looks good.

**Review by vrinda-mahajan — CHANGES_REQUESTED** [review]

File: css/login.css [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+.title-login,
+.link-account {
+    text-align: center;
+}
+
+.form-checkbox {
+display: flex;
+justify-content: space-between;
+align-items: center;
+}
```
Comment (anik31) [inline css/login.css]: Add indentation

**Comment by anik31** [comment]
Add images in PR description so that we can review the UI as well.

**Comment by mikesch803** [comment]
fit-store-login
fit-store-signup

File: css/login.css (hunk 6 above)
Comment (mikesch803) [reply css/login.css]: fix it

File: css/login.css (hunk 5 above)
Comment (mikesch803) [reply css/login.css]: added the semicolon

**Review by anik31 — APPROVED** [review]
UI looks nice. Good work

**Comment by sainath7878** [comment]
Looks good to me.

**Review by sainath7878 — APPROVED** [review]

**Pull request merged**